### PR TITLE
Add gfxdata attribute 

### LINKED ISSUE #709: Schema Error : The 'urn:schemas-microsoft-com:office:office:gfxdata' attribute is not declared.
Is this a:
- [X] Issue with the OpenXml library
- [ ] Question on library usage

**Description**

Schema Error : The 'urn:schemas-microsoft-com:office:office:gfxdata' attribute is not declared. On 2 separate valid Office 365 Word documents.

**Information**

- .NET Target: .NET 4.6.2
- DocumentFormat.OpenXml Version: 2.10.1

**Repro**

Validate the attached file with a WordDocumentValidator.

**Observed**

The following validation error was reported.

Schema Error : The 'urn:schemas-microsoft-com:office:office:gfxdata' attribute is not declared.

**Expected**

The file should have been validated successfully without errors.
[gfxdata.docx](https://github.com/OfficeDev/Open-XML-SDK/files/4383497/gfxdata.docx)

### PR DESCRIPTION
**Pull request by tomjebo**
add gfxdata attribue to Arc, Curve, Line, PolyLine, Group, Image, Oval, Rect, and RoundRect shape complex types per MS-OI29500 2.1.1783-1799

Fixes #709

### CODE REVIEW

**Comment by tomjebo** [comment]
Assert has been corrected per the usage style. However, let's hold off on merging just until I resolve the question on the schema replace in the backend.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added `OpenXmlElementFunctionalExtensions.With(...)` extension methods, which offer flexible means for constructing `OpenXmlElement` instances in the context of pure functional transformations (#679)
 - Added minimum Office versions for enum types and values (#707).
 - Added additional `CompatSettingNameValues` values: `UseWord2013TrackBottomHyphenation`, `AllowHyphenationAtTrackBottom`, and `AllowTextAfterFloatingTableBreak` (#706).
+- Added gfxdata attribue to Arc, Curve, Line, PolyLine, Group, Image, Oval, Rect, and RoundRect shape complex types per MS-OI29500 2.1.1783-1799 ($709)
```
Comment (twsouthwick) [inline CHANGELOG.md:14]: Should be a pound sign. `#709` not `$709`

File: src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_vml.g.cs [hunk 2]
```diff
@@ -3184,7 +3195,7 @@ public Group(string outerXml) : base(outerXml)
         /// xmlns:o=urn:schemas-microsoft-com:office:office
         /// </remark>
         [SchemaAttr(27, "tableproperties")]
-        [Index(31)]
+        [Index(32)]
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_vml.g.cs:3198]: Some of the last magic numbers in this thing. I hope to get them cleaned up in the next couple weeks once my workload lessens a bit.

Comment (tomjebo) [reply src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_vml.g.cs:3198]: Did you need something changed here? Is the index number changing causing a problem?

Comment (tomjebo) [reply src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_vml.g.cs:3198]: @twsouthwick or can we proceed with the merge?

Comment (twsouthwick) [reply src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_vml.g.cs:3198]: Oh - that was just a comment that I've been trying to clear up these magic numbers. no action needed, just pointing out where these are for future information :)

Comment (tomjebo) [reply src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_vml.g.cs:3198]: got it. yeah, I noticed them in the last two changes. I was worried at first that they may cause conflicts or issues but apparently they don't. We should add a workitem in the backend.

**Pull request merged**